frontend/src/components/portforward/index.tsx

```diff
@@ -0,0 +1,231 @@
+
+        // remove portforward from storage too
+        const portforwardInStorage = localStorage.getItem(PORT_FORWARDS_STORAGE_KEY);
+        const parsedPortForwards = JSON.parse(portforwardInStorage || '[]');
+        const index = parsedPortForwards.findIndex((pf: any) => pf.id === id);
+        if (index !== -1) {
+          parsedPortForwards.splice(index, 1);
+        }
+        localStorage.setItem(PORT_FORWARDS_STORAGE_KEY, JSON.stringify(parsedPortForwards));
+
+        // update portforward list item
+        fetchPortForwardList(true);
+      });
+    }
+  };
+
+  function prepareStatusLabel(portforward: any) {
+    if (portForwardInAction?.id === portforward.id && portForwardInAction.loading) {
+      return <Loader noContainer title="Loading" size={30} />;
+    }
+    const error = portforward.error;
+    if (error) {
+      return <StatusLabel status="error">Error</StatusLabel>;
+    }
+    return (
+      <StatusLabel
+        status={portforward.status === PORT_FORWARD_RUNNING_STATUS ? 'success' : 'error'}
+      >
+        {portforward.status}
+      </StatusLabel>
+    );
+  }
+  return (
+    <SectionBox title="Port Forwards">
+      <SimpleTable
+        columns={[
+          {
+            label: 'Name',
+            getter: portforward => {
+              const podOrService = portforward.service ? 'service' : 'pod';
+              const name = portforward.service || portforward.pod;
+              const namespace = portforward.serviceNamespace || portforward.namespace;
+              return (
+                <Link routeName={podOrService} params={{ name, namespace }}>
+                  {name}
+                </Link>
+              );
+            },
+          },
+          {
+            label: 'Namespace',
+            getter: portforward => {
+              return portforward.serviceNamespace || portforward.namespace;
+            },
+          },
+          {
+            label: 'Kind',
+            getter: portforward => {
+              return portforward.service ? 'Service' : 'Pod';
+            },
+          },
+          {
+            label: 'Pod Port',
+            getter: portforward => {
+              return portforward.targetPort;
+            },
+          },
+          {
+            label: 'Local Port',
+            getter: portforward => {
+              return portforward.port;
+            },
+          },
+          {
+            label: 'Status',
+            getter: portforward => {
+              return prepareStatusLabel(portforward);
+            },
+          },
+          {
+            label: 'Actions',
+            getter: portforward => {
+              const filteredOptions = options.filter((option: any) => {
+                if (portForwardInAction?.error) {
+                  return option === 'Delete';
+                }
+                if (portForwardInAction?.status === PORT_FORWARD_RUNNING_STATUS) {
+                  return option !== 'Start';
```

Comment:
There are lots of strings that need to be translatable in this UI, including the ops, and when that's done, this logic needs to be changed.